client/src/components/latestCampaigns/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Typography, Anchor } from 'antd';
+import Campaign from '../common/campaignCard';
+import './style.less';
+
+const { Link } = Anchor;
+const { Title, Text } = Typography;
+function latestCampaigns({ campaigns }) {
+  const toCampaignsPage = (e) => {
+    e.preventDefault();
+    // Navigate to /Campaigns
+  };
+  return (
+    <section className="layout">
+      <Title className="header_title" level={1}>
+        <Text>
+          Latest
+        </Text>
+        Campaigns
+      </Title>
+      <div className="cards">
+        {
+          // campaigns is only three campagins
+          campaigns.map((({
+            id,
+            title,
+            description,
+            imgSrc,
+            categoryIcon,
+          }) => (
+            <Campaign
+              key={id}
+              title={title}
+              description={description}
+              imgSrc={imgSrc}
+              categoryIcon={categoryIcon}
+            />
```

Comment:
You need to edit the card so when you click on it, it will link you to this campaign page